lib/battle_snake/snake.ex

```diff
@@ -1,5 +1,12 @@
+defimpl Poison.Encoder, for: BattleSnake.Snake do
+  def encode(snake, opts) do
+    keys = [:coords, :id, :taunt, :health_points, :name] ++
+      if Keyword.get(opts, :consumer) do  [ :head_url, :color] else [] end
+    Poison.encode!(Map.take(snake, keys))
+  end
+end
+
```

Comment:
Yeah, I tried setting opts to just :consumer, in World.encode, but other stuff blew up. You're not shy about pasting extra lines, eh?  Ok, expanded.